chainer/functions/normalization/l2_normalization.py

```diff
@@ -43,7 +67,11 @@ def backward(self, indexes, grad_outputs):
         norm = F.broadcast_to(norm, gy.shape)
 
         x_gy_reduced = F.sum((x * gy), axis=self.axis, keepdims=True)
-        x_gy_reduced /= norm_noeps
+        mask = norm_noeps.array != 0
```

Comment:
Please add brief comment to describe that the intention is to avoid zero division.